Source/Scene/Cesium3DTileBatchTableResources.js

```diff
@@ -2,14 +2,19 @@
 define([
         '../Core/arrayFill',
         '../Core/Cartesian2',
+        '../Core/Cartesian3',
```

Comment:
`Cesium3DFeatureTable` is also `Cesium3DFeatureTableResources`. I agree that it's worth making both more concise.